examples/tests/sdk_tests/std/json.test.w

```diff
@@ -178,4 +178,25 @@ test "deepCopy(), deepCopyMut()" {
   assert(copy != copyMut);
 
   assert(copyMut.get("object") == mutation);
+}
+
+test "delete() for MutJson" {
+  let mutObj = MutJson { x: 1, y: 2 };
+  mutObj.delete("x");
+  let assertThrows = (expected: str, block: (): void) => {
```

Comment:
I think you can just use `Json.has(x)` instead to verify that the key was deleted. No?

Reply:
I don't think I can since the has() method doesn't exist for the MutJson class from what I understand

Reply:
has() method added to MutJson class as part of this PR!